api/tools/src/main/java/org/pmiops/workbench/tools/CreateTerraMethodSnapshot.java

```diff
@@ -250,9 +248,6 @@ private String getGithubFileContents(
   }
 
   public static void main(String[] args) {
-    new SpringApplicationBuilder(CreateTerraMethodSnapshot.class)
-        .web(WebApplicationType.NONE)
-        .run(args)
-        .close();
+    CommandLineToolConfig.runCommandLine(CreateTerraMethodSnapshot.class, args);
```

Comment:
figured this one out thanks to a 5-year-old comment from CH in ManageLeonardoRuntimes!